easybuild/easyconfigs/a/ABINIT/ABINIT-9.2.1-foss-2019b.eb

```diff
@@ -0,0 +1,56 @@
+#
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = '9.2.1'
+
+homepage = 'https://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy,
+ charge density and electronic structure of systems made of electrons and nuclei (molecules
+ and periodic solids) within Density Functional Theory (DFT), using pseudopotentials and a
+ planewave or wavelet basis."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://www.abinit.org/sites/default/files/packages/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['4aa2deaeec385ff1624669a59768e1a6655f6367f8f109e69944244e000142a0']
+
+dependencies = [
+    ('libxc', '4.3.4'),
+    ('netCDF-Fortran', '4.5.2'),
+]
+
+# Ensure MPI.
+configopts = '--with-mpi="yes" --enable-openmp="no" '
+
+# BLAS/Lapack
+configopts += '--with-linalg-flavor="openblas"  LINALG_LIBS="-L${EBROOTOPENBLAS}/lib -lopenblas" '
+
+# FFTW3 support
+configopts += '--with-fft-flavor=fftw3 FFTW3_LIBS="-L${EBROOTFFTW} -lfftw3f -lfftw3" '
+
+# libxc support
+configopts += '--with-libxc=${EBROOTLIBXC} '
+
+# hdf5/netcdf4.
+configopts += 'with_netcdf="${EBROOTNETCDF}" '
+configopts += 'with_netcdf_fortran="${EBROOTNETCDFMINFORTRAN}" '
+configopts += 'with_hdf5="${EBROOTHDF5}" '
```

Comment:
```suggestion
# hdf5/netcdf4 support
configopts += '--with-netcdf="${EBROOTNETCDF}" '
configopts += '--with-netcdf-fortran="${EBROOTNETCDFMINFORTRAN}" '
configopts += '--with-hdf5="${EBROOTHDF5}" '
```

Reply:
Thanks for spotting the error. 
I've inadvertently used the normalized name used when passing options via an external file instead of the command line options.